client/magic-link.ts

```diff
@@ -134,7 +134,7 @@ function checkCurrentLocationForSignInLink() {
     debug?.("Magic link header parsed:", message);
     assertIsMessage(message);
   } catch (err) {
-    debug?.("Ignoring invalid fragment identifier");
+    debug?.("Ignoring invalid fragment identifier", { err });
```

Comment:
Nit: Conscious of `err` being some huge object, with unhelpful stack traces, or whatever, could we log out a message from within `err`, perhaps?